### DISCUSSION TITLE
Class "TomatoPHP\TomatoPHP\Services\Menu\TomatoMenuRegister" not found

### DISCUSSION THREAD

**Discussion by ouziine** [tomatophp/tomato#20] [Show and tell]
i get this error:
**Class "TomatoPHP\TomatoPHP\Services\Menu\TomatoMenuRegister" not found**
how can i solve this problem?

**Comment by fadymondy** [comment]
!Don't update any old project to v1.1 

we changed some main classes to be on tomato-admin so the namespace has been changed

**Reply by ahadyt555** [reply]
hi bro whould you have any installation video of tomato php in installation guide its not a proper article for installation i am trying to get tomato.php, but i cant

**Reply by fadymondy** [reply]
Good idea, i will create one

**Comment by CEVR88** [comment]
Is any update or way to fix this error that I am facing too?

**Reply by fadymondy** [reply]
yes just update to v1.2 of tomato-admin!
